src/scss/global.scss

```diff
@@ -551,12 +551,10 @@ h4 {
       bottom: 10px;
       left: 0;
       right: 0;
-      text-align: center;
-      display: flex;
-      justify-content: center;
     }
 
     .mute-other-container {
+      cursor: pointer;
```

Comment:
still not seeing pointer on my venue (JazzBar):
http://localhost:3000/in/viktoryiasmusicbar

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2221/commits/0f5a75531713d54e6a9146c2351117298091020d